neurobooth_os/main_control_rec.py

```diff
@@ -66,11 +66,6 @@ def prepare_devices(collection_id="mvp_025", nodes=("acquisition", "presentation
         socket_message(f"prepare:{collection_id}", node)
 
 
-def task_presentation(task_name, filename, node):
```

Comment:
this is a really bad function because of two reasons:
1) it encapsulates a one-line function (should be at least 3 lines to be considered a function)
2) the name is not descriptive of what it does --> printing message
best to get rid of it